ui/components/header-footer.js

```diff
@@ -70,6 +70,8 @@ export default function HeaderFooter({ children, showSearch }) {
       <Head>
         <title>OMB Policy Library (Beta)</title>
         <link rel="stylesheet" href="/static/styles.css" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.css" integrity="sha384-8QOKbPtTFvh/lMY0qPVbXj9hDh+v8US0pD//FcoYFst2lCIf0BmT58+Heqj0IGyx" crossOrigin="anonymous" />
```

Comment:
We may be able to [import this](https://github.com/18F/omb-eregs/blob/da8c484254201380ea2a577152642d30089525f6/ui/css/main.scss#L7-L8) into our styles from the npm library, which would allow us to bundle the CSS rather than make another HTTP call.

Reply:
I ran into font import errors when trying this, so I'm going to leave it as is for the moment.